Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -503,4 +429,76 @@ namespace Multiplayer
         }
     }
 
-}
+    void MultiplayerEditorSystemComponent::OnPreparingSpawnable(const AzFramework::Spawnable& spawnable, const AZStd::string& assetHint)
+    {
+        // Only grab the level (Root.spawnable or Root.network.spawnable)
+        // We'll receive OnPreparingSpawnable for other spawnables that are referenced by components in the level,
+        // but these spawnables are already available for the server inside the asset cache.
+        if (assetHint != "Root.spawnable" && assetHint != "Root.network.spawnable")
```

Comment:
It's not inconceivable that down the road other types of spawnables besides a basic and network spawnable are created. (Things like tiles come to mind.) It would perhaps add some future proofing if the check here is for any asset that starts with `"Root."`. It would also be good to move the name "Root" in a centralized variable because it's hardcoded a few times already, which makes it difficult to change in the future.

Reply:
Good call! Updated... moved DefaultMainSpawnableName out of AzToolsFramework into AzFramework::Spawnable along side only useful spawnable consts.